<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="prosesLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prosesButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Proses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="actionOperasi_Pencerah"/>
    <addaction name="actionOperasi_Kontras"/>
    <addaction name="actionPeregangan_Kontras"/>
    <addaction name="actionNegative"/>
    <addaction name="actionBiner_Image"/>
   </widget>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="actionHistogram_Grayscale"/>
    <addaction name="actionHistogram_RGB"/>
    <addaction name="actionHistogram_Equalization"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Geometri">
    <property name="title">
     <string>Operasi Geometri</string>
    </property>
    <widget class="QMenu" name="menuRotasi">
     <property name="title">
      <string>Rotasi</string>
     </property>
     <addaction name="action_90"/>
     <addaction name="action_45"/>
     <addaction name="action45"/>
     <addaction name="action90"/>
     <addaction name="action180"/>
    </widget>
    <widget class="QMenu" name="menuResize">
     <property name="title">
      <string>Resize</string>
     </property>
     <addaction name="actionZoom_In"/>
     <addaction name="actionZoom_Out"/>
     <addaction name="actionSkewed"/>
    </widget>
    <addaction name="actionTranslasi_Citra"/>
    <addaction name="menuRotasi"/>
    <addaction name="menuResize"/>
    <addaction name="actionCrop"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Aritmatika">
    <property name="title">
     <string>Operasi Aritmatika</string>
    </property>
    <addaction name="actionTambah_dan_Kurang"/>
   </widget>
   <widget class="QMenu" name="menuBoolean">
    <property name="title">
     <string>Boolean</string>
    </property>
    <addaction name="actionAnd"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Spasial">
    <property name="title">
     <string>Operasi Spasial</string>
    </property>
    <widget class="QMenu" name="menuImage_Shapening">
     <property name="title">
      <string>Image Shapening</string>
     </property>
     <addaction name="actionI"/>
     <addaction name="actionII"/>
     <addaction name="actionIII"/>
     <addaction name="actionIV"/>
     <addaction name="actionV"/>
     <addaction name="actionVI"/>
     <addaction name="actionFilter_Laplace"/>
    </widget>
    <widget class="QMenu" name="menuKonvolusi_2D">
     <property name="title">
      <string>Konvolusi 2D</string>
     </property>
     <addaction name="actionA"/>
     <addaction name="actionB"/>
    </widget>
    <widget class="QMenu" name="menuMean_Filter">
     <property name="title">
      <string>Mean Filter</string>
     </property>
     <addaction name="actionI_2"/>
     <addaction name="actionII_2"/>
    </widget>
    <addaction name="menuKonvolusi_2D"/>
    <addaction name="menuMean_Filter"/>
    <addaction name="actionGausian_Filter"/>
    <addaction name="menuImage_Shapening"/>
    <addaction name="actionMax_Filter"/>
    <addaction name="actionMedian_Filter"/>
   </widget>
   <addaction name="menuOperasi_Titik"/>
   <addaction name="menuHistogram"/>
   <addaction name="menuOperasi_Geometri"/>
   <addaction name="menuOperasi_Aritmatika"/>
   <addaction name="menuBoolean"/>
   <addaction name="menuOperasi_Spasial"/>
  </widget>
  <action name="actionOperasi_Pencerah">
   <property name="text">
    <string>Operasi Pencerah</string>
   </property>
  </action>
  <action name="actionOperasi_Kontras">
   <property name="text">
    <string>Pengaturan Kontras</string>
   </property>
  </action>
  <action name="actionPeregangan_Kontras">
   <property name="text">
    <string>Peregangan Kontras</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionBiner_Image">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
  <action name="actionHistogram_Grayscale">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="actionHistogram_RGB">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="actionHistogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionTranslasi_Citra">
   <property name="text">
    <string>Translasi Citra</string>
   </property>
  </action>
  <action name="action_45_Derajat">
   <property name="text">
    <string>-45 Derajat</string>
   </property>
  </action>
  <action name="action45_Derajat">
   <property name="text">
    <string>45 Derajat</string>
   </property>
  </action>
  <action name="action_90_Derajat">
   <property name="text">
    <string>-90 Derajat</string>
   </property>
  </action>
  <action name="action90_Derajat">
   <property name="text">
    <string>90 Derajat</string>
   </property>
  </action>
  <action name="action180_Derajat">
   <property name="text">
    <string>180 Derajat</string>
   </property>
  </action>
  <action name="action_90">
   <property name="text">
    <string>-90 Derajat</string>
   </property>
  </action>
  <action name="action_45">
   <property name="text">
    <string>-45 Derajat</string>
   </property>
  </action>
  <action name="action45">
   <property name="text">
    <string>45 Derajat</string>
   </property>
  </action>
  <action name="action90">
   <property name="text">
    <string>90 Derajat</string>
   </property>
  </action>
  <action name="action180">
   <property name="text">
    <string>180 Derajat</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionSkewed">
   <property name="text">
    <string>Skewed</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionTambah_dan_Kurang">
   <property name="text">
    <string>Tambah dan Kurang</string>
   </property>
  </action>
  <action name="actionAnd">
   <property name="text">
    <string>And</string>
   </property>
  </action>
  <action name="actionGausian_Filter">
   <property name="text">
    <string>Gausian Filter</string>
   </property>
  </action>
  <action name="actionI">
   <property name="text">
    <string>I</string>
   </property>
  </action>
  <action name="actionII">
   <property name="text">
    <string>II</string>
   </property>
  </action>
  <action name="actionIII">
   <property name="text">
    <string>III</string>
   </property>
  </action>
  <action name="actionIV">
   <property name="text">
    <string>IV</string>
   </property>
  </action>
  <action name="actionV">
   <property name="text">
    <string>V</string>
   </property>
  </action>
  <action name="actionVI">
   <property name="text">
    <string>VI</string>
   </property>
  </action>
  <action name="actionMax_Filter">
   <property name="text">
    <string>Max Filter</string>
   </property>
  </action>
  <action name="actionMedian_Filter">
   <property name="text">
    <string>Median Filter</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>A</string>
   </property>
  </action>
  <action name="actionB">
   <property name="text">
    <string>B</string>
   </property>
  </action>
  <action name="actionI_2">
   <property name="text">
    <string>I</string>
   </property>
  </action>
  <action name="actionII_2">
   <property name="text">
    <string>II</string>
   </property>
  </action>
  <action name="actionFilter_Laplace">
   <property name="text">
    <string>Filter Laplace</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
